2023-MRC-BSU-and-UKHSA/getting-started.md

```diff
@@ -1,14 +1,7 @@
 # Getting started
 
-These workshops are intended to be somewhat interactive, and so you might be at risk of having to actually write and execute some code on your own device!
-
-If you're already familiar with Julia and have Julia 1.8 or greater already installed, then you can probably stop reading here.
```

Comment:
Made short enough that this is irrelevant -- if people have Julia installed, they can quickly skim and be done with it.